### DISCUSSION TITLE
osu!lazer is crashing :(

### DISCUSSION THREAD

**Discussion by tatitatai** [ppy/osu#20434] [Q&A]
before latest update already, so it can't be about that. mostly crashes after i clicked anything, sometimes even before that. 

i assumed i couldn't update ingame, so i've re-installed completely but it's still happening sadly.

[legacy-ipc.log](https://github.com/ppy/osu/files/9636065/legacy-ipc.log)
[performance.log](https://github.com/ppy/osu/files/9636066/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9636067/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9636068/updater.log)
[database.log](https://github.com/ppy/osu/files/9636069/database.log)
[debug.log](https://github.com/ppy/osu/files/9636070/debug.log)

**Comment by peppy** [comment]
Your logs don't show any errors, so this is looking to be a GPU driver level issue. Can you check event viewer for any crashes in AMD or NVIDIA dll files?

**Comment by tatitatai** [comment]
is this what we're looking for? 

> Unable to recover from a kernel exception. The application must close.
> 
> Error code: 3 (subcode 7)
>  (pid=18948 tid=8896 osu!.exe 64bit)
> 
> Visit http://www.nvidia.com/page/support.html for more information.

or (apparently in german)

> Name der fehlerhaften Anwendung: osu!.exe, Version: 0.0.0.0, Zeitstempel: 0x61e1d8d9
> Name des fehlerhaften Moduls: nvoglv64.dll, Version: 30.0.14.9649, Zeitstempel: 0x617068e7
> Ausnahmecode: 0xc0000409
> Fehleroffset: 0x00000000012820e5
> ID des fehlerhaften Prozesses: 0x3e04
> Startzeit der fehlerhaften Anwendung: 0x01d8cf6ed4b63a5b
> Pfad der fehlerhaften Anwendung: C:\Users\tatjana.vedder\AppData\Local\osulazer\app-2022.923.1\osu!.exe
> Pfad des fehlerhaften Moduls: C:\WINDOWS\System32\DriverStore\FileRepository\nvmdi.inf_amd64_3bab16edc4978b17\nvoglv64.dll
> Berichtskennung: 3a6038d3-1f0d-479e-b87b-3f05eebb9b84
> Vollständiger Name des fehlerhaften Pakets: 
> Anwendungs-ID, die relativ zum fehlerhaften Paket ist:

**Reply by peppy** [reply]
Yep! Can you try disabling nvidia recording overlays and see if it helps?

**Reply by tatitatai** [reply]
thanks, turning it off solved it for me! may i ask how you found out from the log? >_>

**Reply by peppy** [reply]
It's just an error we've seen come up a log so I'm familiar with the likely cause.
